src/lib.rs

```diff
@@ -586,5 +586,6 @@ CREATE AGGREGATE vector_selector(
     deserialfunc = vector_selector_deserialize,
     parallel = safe
 );
-"#
+"#,
+    name = "create_aggregate_vector_selector"
```

Comment:
oh, I forgot about these!
It won't really matter until you start using the pgx schema generator, but you should add
```suggestion
    name = "create_aggregate_vector_selector",
    requires = [vector_selector_transition, vector_selector_final, vector_selector_combine, vector_selector_serialize, vector_selector_deserialize],
```
so the generator outputs the SQL in the correct order.

Reply:
Yes, thanks. I will address this when I get to the schema generator.